tests/devices/test_default_qutrit.py

```diff
@@ -792,7 +940,23 @@ def test_apply_single_qutrit_op(self, op, method, inverse):
         state_out_einsum = np.einsum("ab,ibjk->iajk", matrix, self.state)
         assert np.allclose(state_out, state_out_einsum)
 
-    # TODO: Add tests for two-qutrit operations
+    @pytest.mark.parametrize("op, method", two_qutrit_ops)
+    def test_apply_two_qutrit_op(self, op, method, inverse):
+        """Test if the application of two qutrit operations is correct."""
+        state_out = method(self.state, axes=[0, 1], inverse=inverse)
+        op1 = op(wires=[0, 1])
+        matrix = op1.inv().matrix() if inverse else op1.matrix()
+        matrix = matrix.reshape((3, 3, 3, 3))
+        state_out_einsum = np.einsum("abcd,cdjk->abjk", matrix, self.state)
+        assert np.allclose(state_out, state_out_einsum)
+
+        # Test reversed wire order to see if axes are stacked correctly
```

Comment:
Recommend to make the reverse version a separate test, as is done for the `DefaultQubit` tests.

Reply:
I can't find this code block anymore. I might've updated the tests before the summer ended and can't recall.